src/pages/Admin/organizations/AdminOrganizationView.tsx

```
@@ -153,60 +217,146 @@ export default function AdminOrganizationView({ id, organizationType }: Props) {
+                        {t("actions")}
+                      </TableHead>
+                    </TableRow>
+                  </TableHeader>
+                  <TableBody>
+                    {children.results.map((org) => {
+                      const canDelete = id ? true : !org.has_children;
+                      return (
+                        <TableRow
+                          key={org.id}
+                          tabIndex={0}
+                          role="link"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<a>
<link>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)